sql/sys_vars.cc

```diff
@@ -125,8 +125,8 @@
 #include "sql/sp_head.h"          // SP_PSI_STATEMENT_INFO_COUNT
 #include "sql/sql_backup_lock.h"  // is_instance_backup_locked
 #include "sql/sql_lex.h"
-#include "sql/sql_locale.h"     // my_locale_by_number
-#include "sql/sql_parse.h"      // killall_non_super_threads
+#include "sql/sql_locale.h"  // my_locale_by_number
+#include "sql/sql_parse.h"   // killall_non_super_threads
```

Comment:
if done by clang format, leave it as is. If not, please revert it.

Reply:
> if done by clang format, leave it as is. If not, please revert it.

Done by clang format

Reply:
:+1: